### PR TITLE
Misc fixes and tweaks

### LINKED ISSUE #5694: (WEB REPORT BY: deadpoolj REMOTE: 198.245.63.50:7777) Multiple people can ride in a wheelchair.
>Revision (Should be above if you're viewing this from ingame!)
497bf564a6bf764f4a81cff18982d6177ac931db 

>General description of the issue
Multiple people can ride in a wheelchair at once.

>What you expected to happen
Rambling to bust into my office via AI.

>What actually happened
5 shitters busted into my office.

>Steps to reproduce if possible
Have one nigga in a wheelchair. Have another person stand by it and drag themselves onto the wheelchair. Try stopping the wheelchair tide. Try not to cry. Cry some more.

### PR DESCRIPTION
**Pull request by unid15**
- Fixed chplain's bible being incorrect
- Removed fun by fixing wheelchairs (Fixes #5694)
- Added an admin-only "multi_people" wheelchair which works just like the old wheelchair (with a bonus of actually stacking people on top of each other)
- The "Dude's mind is too faint..." message no longer shows up if Dude is on the second z-level
- Absolute pathing in gun.dm

### CODE REVIEW

File: code/game/jobs/job/civilian_chaplain.dm [hunk 1]
```diff
@@ -216,7 +216,7 @@
 				J = "Trial Admin"
 				deity_name = "The Adminbus"
 			if("coding", "coder", "coders")
-				B.name = "Guide to GitLab"
+				B.name = "Guide to Github"
```
Comment (N3X15) [inline code/game/jobs/job/civilian_chaplain.dm]: Or just "Guide to Git"

**Comment by N3X15** [comment]
Changelog

**Comment by Probe1** [comment]
:+1: 

people might be a bit unhappy about the wheelchair thing but it was weird.  

removing the ability to spy out nuke ops (or more benignly, test dummies in the thunderdome) is top notch work

**Comment by Unfit** [comment]
@N3X15 changelog yes

**Pull request merged**